bin/interpreter.sh

```diff
@@ -129,6 +129,7 @@ if [[ "${INTERPRETER_ID}" == "spark" ]]; then
 
     export SPARK_CLASSPATH+=":${ZEPPELIN_CLASSPATH}"
   fi
+  CLASSPATH+=":${ZEPPELIN_CLASSPATH}":"${ZEPPELIN_HOME}/interpreter/spark/R/rscala_2.10-1.0.6.jar"
```

Comment:
How does this intepreter.sh line relate to the profile selected during the build?
I.e., you recently added the ability to include a build profile such as `-Drscala.version=1.0.8` or `-Drscala.version=1.0.6` etc